scss/civicrm/cms/_drupal.scss

```diff
@@ -0,0 +1,109 @@
+/* stylelint-disable selector-max-id */
+
+@import 'base/scss/vendor/bootstrap/alerts';
+
+.page-civicrm {
+  %title {
+    font-weight: 600 !important;
+  }
+
+  h2 {
+    @extend %title;
+  }
+
+  #branding {
+    background: $crm-page-topbar-bg !important;
+    color: $crm-white !important;
+    font-family: $font-family-base;
+    line-height: 20px;
+    overflow: visible !important;
+    padding: 20px 0 0;
+
+    h1.page-title { /* stylelint-disable-line selector-no-qualifying-type */
+      @extend %title;
+
+      background: $crm-white;
+      box-sizing: border-box;
+      color: $gray-darker;
+      display: block;
+      line-height: 50px !important;
+      margin: -20px 0 20px;
```

Comment:
same as above.
also, there are other places in this PR where you use 20, 40 etc, if it relates to the "grid" may be we should use the '$crm-standard-gap' as well?